kafka-postgres-reader/src/main/java/io/arenadata/kafka/postgres/reader/controller/QueryController.java

```diff
@@ -56,6 +56,7 @@ public void query(RoutingContext context) {
                     })
                     .onFailure(t -> {
                         log.error("Query execute FAILED", t);
+                        context.response().setStatusMessage(t.getMessage());
```

Comment:
You checked that behaivor? 
`context.fail(t);` is not rewriting that status message?

Reply:
Yup. DTM gets the message as it's set on this line. Any concerns on this?